mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -169,7 +187,10 @@ where
                             self.handle_unsubscribed(&unsub);
                         }
                         SubscriptionUpdateEvent::RejectedByServer(sub) => {
-                            debug!("received subscription rejected: {}", sub.topic_filter);
+                            warn!(
```

Comment:
Can we also add which server?

Reply:
doesn't have here the details about the server, changed to broker which now it must be upstream